fractal_server/app/runner/_slurm/__init__.py

```diff
@@ -19,6 +25,75 @@
 """
 
 
+class SlurmConfig(BaseModel):
+    """
+    `name` maps on `Task.executor`.
```

Comment:
`name` is not in the list of class attributes. Is this on purpose?

Reply:
Ok, now I see that `SlurmConfig.name` was moved to the keys of `config_dict` (below).

Then we only need to update the docstring.

Reply:
BTW, we always want to use `WorkflowTask.executor`, and not `Task.executor` (it's only wrong in the docstring)